### ISSUE TITLE
Does maskrcnn benchmark support half precision inference?

### ISSUE THREAD

**Issue by wlguan** [facebookresearch/maskrcnn-benchmark#888]
## ❓ Questions and Help
Does maskrcnn benchmark support half precision inference? If not, what should I add?

**Comment by miguelvr** [comment]
yes, it does

**Comment by wlguan** [comment]
> yes, it does
> 
> EDIT: not sure about inference, but it definitely supports training

Yes, I notice the DTYPE option. Hope that half-precision inference can also be supported.
